src/Symfony/Bundle/FrameworkBundle/Translation/PhpExtractor.php

```diff
@@ -20,7 +21,7 @@
  *
  * @author Michel Salib <michelsalib@hotmail.com>
  */
-class PhpExtractor implements ExtractorInterface
+class PhpExtractor extends AbstractFileExtractor implements ExtractorInterface
```

Comment:
Does it make sense to let `AbstractFileExtractor` implement `ExtractorInterface` instead? Or can we think about a use case where this is not necessary?